packages/tools/client-debugger/client-debugger-view/src/components/WaitingLabels.tsx

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+/**
+ * Type of {@link waitingLabels}.
+ */
+export type WaitingLabelsType = Record<string, string>;
+
+/**
+ * Record containing labels for {@link Waiting} component.
+ */
+export const waitingLabels: WaitingLabelsType = {
+	containerError: "Waiting for container DDS data.",
+	undefinedError: "Data undefined.",
```

Comment:
These cases don't correspond to "waiting". They're "error" text we include in components. And since each of these is specific to a single component type, I think we can just define them in the corresponding component's file.